package.json

```diff
@@ -68,21 +68,11 @@
         "command": "tacosontitan.toolbox.azure.devops.createDefaultTasks",
         "title": "Hazel's Toolbox: Create Default Azure DevOps Tasks"
       },
-      {
-        "command": "tacosontitan.toolbox.azure.devops.setWorkItem",
-        "title": "Hazel's Toolbox: Set Work Item",
-        "icon": "$(search)"
-      },
       {
         "command": "tacosontitan.toolbox.azure.devops.refreshTasks",
         "title": "Hazel's Toolbox: Refresh",
         "icon": "$(refresh)"
       },
-      {
-        "command": "tacosontitan.toolbox.azure.devops.addTask",
-        "title": "Hazel's Toolbox: Add Task",
```

Comment:
We still want to be able to add a task to any of the top-level work items in the tree, so we should keep this around.